### DISCUSSION TITLE
Plugin Idea: Twitch Emotes

### DISCUSSION THREAD

**Discussion by mohcow** [Flow-Launcher/Flow.Launcher#1001] [Ideas]
Hi,

As an avid Twitch viewer it would be cool to be able to search for an image link of a specific emote like Emoji+ does it. You could query the user input to `https://betterttv.com/emotes/shared/search?query=%s` and show the results in Flow Launcher. After choosing a result add the image address f.e. `https://cdn.betterttv.net/emote/5ef2291ef91de70dea5be480/3x` to the clipboard. The best way would probably to query them directly instead of caching since new ones get added all the time, but im a layman so you do you.

**Comment by Garulf** [comment]
Looking into it and the API seems pretty straight forward.

![image](https://user-images.githubusercontent.com/535299/151761460-3a72190e-0896-481c-a3b8-837367b92d89.png)

Edit: their CDN is incredibly fast

**Comment by Garulf** [comment]
Plugin is done and submitted to our plugin store.

Submission: https://github.com/Flow-Launcher/Flow.Launcher.PluginsManifest/pull/135
Github page: https://github.com/Garulf/BetterTTV-Twitch-Emotes

Once the submission is approved and added you can type:
```
pm install betterttv twitch emotes
```

Or install directly on FLow Launcher 1.9.0 or higher by typing:
```
pm install https://github.com/Garulf/BetterTTV-Twitch-Emotes/releases/download/v0.0.1/BetterTTV-Twitch-Emotes.zip
```
